op-node/rollup/clsync/clsync.go

```diff
@@ -104,77 +104,46 @@ func (eq *CLSync) onInvalidPayload(x engine.PayloadInvalidEvent) {
 // onForkchoiceUpdate peeks at the next applicable unsafe payload, if any,
```

Comment:
We may update this comment like
```suggestion
// onForkchoiceUpdate refreshes unsafe payload queue and peeks at the next applicable unsafe payload, if any,
```